app/sass/login-form.scss

```diff
@@ -0,0 +1,72 @@
+$base-color: #b73535;
+@mixin login-form-font($size) {
+  font: {
+    family: Georgia, Cambria, "Times New Roman", Times, serif;
+    style: italic;
+    size: $size;
+  };
+}
+.login-form {
+  margin-top: 100px;
+  display: flex;
+  justify-content: center;
+  input {
+    background: none;
+    border: solid 2px $base-color;
+    color: $base-color;
+    padding: 15px 40px;
+    font-size: 18px;
+    display: inline-block;
+    &:focus,
+    &:active {
+      outline: none;
+    }
+    &[type="text"],
```

Comment:
This ruleset is not needed. `border` and `boreder-bottom` are inherited from `input` anyway